functions/is-present/1.0/index.js

```diff
@@ -0,0 +1,5 @@
+const isPresent = async ({ anyValue }) => ({
+  result: Array.isArray(anyValue) ? anyValue.length > 0 : !!anyValue,
```

Comment:
This will not check for empty objects. You could do;

`result: !!anyValue && Object.keys(anyValue).length > 0`

Reply:
It does check empty object. I've a test with role object with id equals 0. that record does not exists and the input anyvalue is required. 
Your solution isn't working on empty collections.